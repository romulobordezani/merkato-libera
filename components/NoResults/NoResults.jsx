import React from 'react';
import PropTypes from 'prop-types';

import styles from './NoResults.scss';

const NoResults = () => {
  return (
    <div className={styles.NoResults}>
      <div className={styles.NoResults_icon}>
        <img src="/images/page-not-found.svg" alt="Not Found" />
      </div>
      <div className={styles.NoResults_message}>Ning√∫n resultado para la busqueda.</div>
    </div>
  );
};

NoResults.propTypes = {
  item: PropTypes.object.isRequired
};

export default NoResults;
