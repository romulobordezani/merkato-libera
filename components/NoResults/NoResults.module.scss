.NoResults {
  padding: 16px;
  display: flex;

  &_icon {
    img {
      width: 80px;
      border-radius: 4px;
    }
  }

  &_message {
    font-size: 1.2em;
    padding-left: 16px;
  }
}
