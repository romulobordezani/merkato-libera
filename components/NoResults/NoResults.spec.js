import React from 'react';
import renderer from 'react-test-renderer';

import NoResults from './NoResults';

describe('NoResults', () => {
  it('Snapshot renders default', () => {
    expect(renderer.create(<NoResults />)).toMatchSnapshot();
  });
});
