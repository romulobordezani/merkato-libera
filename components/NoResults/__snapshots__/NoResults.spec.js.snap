// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults Snapshot renders default 1`] = `
<div>
  <div>
    <img
      alt="Not Found"
      src="/images/page-not-found.svg"
    />
  </div>
  <div>
    Ning√∫n resultado para la busqueda.
  </div>
</div>
`;
